aldjemy/orm.py

```diff
@@ -46,9 +46,10 @@ def _extract_model_attrs(model, sa_models):
         p_table = tables[parent_model.db_table]
         p_name = parent_model.pk.column
 
+        disable_backref = True if fk.rel.related_name and fk.rel.related_name.endswith('+') else False
```

Comment:
`a = True if expression else False`
actually its just `a = expression`, or if you need `bool` then use `bool(expression)`